### DISCUSSION TITLE
Dialog Routing approach with /app/ dir

### DISCUSSION THREAD

**Discussion by smo043** [vercel/next.js#46728] [Help] [answered]
I'm using the most recent version of Next JS, and I'd like to see if there's a way to achieve "Dialog/Modal" routing where the current route stays the same and when the "Dialog/Modal" opens, I should have a routing approach without changing the url.

**Comment by guitheengineer** [comment, marked as answer]
If the URL doesn't need to change, just open a client side modal as we usually do.
